lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -201,6 +203,56 @@ def test_send_to_staff(self):
             [self.instructor.email] + [s.email for s in self.staff]
         )
 
+    def test_send_to_cohort(self):
+        """
+        Make sure email sent to a cohort goes there.
+        """
+        cohort = CourseCohort.create(cohort_name='test cohort', course_id=self.course.id)
+        for student in self.students:
+            add_user_to_cohort(cohort.course_user_group, student.username)
+        # Now we know we have pulled up the instructor dash's email view
+        # (in the setUp method), we can test sending an email.
+        test_email = {
+            'action': 'Send email',
+            'send_to': '["cohort:{}"]'.format(cohort.course_user_group.name),
+            'subject': 'test subject for cohort',
+            'message': 'test message for cohort'
+        }
+        # Post the email to the instructor dashboard API
+        response = self.client.post(self.send_mail_url, test_email)
+        # from nose.tools import set_trace; set_trace()
+        self.assertEquals(json.loads(response.content), self.success_content)
+
+        self.assertEquals(len(mail.outbox), len(self.students))
+        self.assertItemsEqual(
+            [e.to[0] for e in mail.outbox],
+            [s.email for s in self.students]
+        )
```

Comment:
`assertEqual` will fail on ordering, `assertItemsEqual` does set comparison and will not throw if two elements have their order swapped. Since the ordering is irrelevant for correctness in this case, I chose `assertItemsEqual`